analysis/tests/src/expected/CompletionFunctionArguments.res.txt

```diff
@@ -33,12 +37,20 @@ Completable: Cexpression CArgument Value[someFn](~isOn)=t
     "tags": [],
     "detail": "bool",
     "documentation": null
+  }, {
+    "label": "typeof",
+    "kind": 12,
+    "tags": [],
+    "detail": "'a => Core__Type.t",
```

Comment:
Notice `Core__Typet` is leaked.

Reply:
CC @zth

Reply:
Yeah, I think we'll need to special case Core. Good catch.